From 2ab0636e0c8fcb8b5b1b222f0d5ae7f4dfc663c5 Mon Sep 17 00:00:00 2001
From: "Michael S. Tsirkin" <mst@redhat.com>
Date: Sat, 5 Oct 2019 17:09:17 -0400
Subject: [PATCH] tests: allow empty expected files

An empty expected file is a handy way to seed the files
without creating merge conflicts.

Signed-off-by: Michael S. Tsirkin <mst@redhat.com>
---
 tests/bios-tables-test.c | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/tests/bios-tables-test.c b/tests/bios-tables-test.c
index a356ac3489..53a91a8067 100644
--- a/tests/bios-tables-test.c
+++ b/tests/bios-tables-test.c
@@ -334,7 +334,10 @@ try_again:
         g_assert(ret);
         g_assert_no_error(error);
         g_assert(exp_sdt.aml);
-        g_assert(exp_sdt.aml_len);
+        if (!exp_sdt.aml_len) {
+            fprintf(stderr, "Warning! zero length expected file '%s'\n",
+                    aml_file);
+        }

         g_array_append_val(exp_tables, exp_sdt);
     }
--
2.19.1
